tests/measurements/test_sample.py

```diff
@@ -319,6 +326,13 @@ def test_shape(self, obs):
         expected = (1, shots) if obs is not None else (1, shots, 3)
         assert res.shape(dev) == expected
 
+    def test_shape_wires(self):
+        """Test that the shape is correct when wires are provided."""
+        shots = 10
```

Comment:
Can we have a fixture for shots and wires and add the tests for `shots==1`,  `wires==1`,